packages/web/components/chart/sparkline.tsx

```diff
@@ -26,7 +27,7 @@ export const Sparkline: FunctionComponent<{
   lineWidth?: number;
   color?: string;
   data: number[];
-}> = ({ width, height, color, data, lineWidth = 1 }) => {
+}> = memo(({ width, height, color, data, lineWidth = 1 }) => {
```

Comment:
Is memo not the default? Curious

Reply:
For vanilla React, it's not. However, every `observer` component does use memo under the hood